MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1069 @@
+
+#### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public string ItemName { get; set; }
+    
+    public class SourceProviderExt : IDataFormSourceProvider
+    {
+        public override IList GetSource(string sourceName)
+        {
+            var list = new List<string>();
+            if (sourceName == "ItemName")
+            {
+                list.Add("Item1");
+                list.Add("Item2");
+                list.Add("Item3");
+            }
+            return list;
+        }
+    }
+    dataForm.ItemsSourceProvider = new SourceProviderExt();
+    dataForm.RegisterEditor("ItemName", DataFormEditorType.ComboBox);
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using ItemsSource property
+
+You can also set the ItemsSource for combo box editor by using the [ItemsSource]`Link` property in the `DataFormComboBoxItem`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            var list = new List<string>();
+            list.Add("Home");
+            list.Add("Food");
+            list.Add("Utilities");
+            list.Add("Education");
+            (e.DataFormItem as DataFormComboBoxItem).ItemsSource = list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Changing ItemsSource of SfComboBox at run time
+
+You can also change the `ItemsSource` at runtime.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    private void Button_Click(object sender, EventArgs e)
+    {
+        var dataFormItem = dataForm.ItemManager.DataFormItems["Name"];
+        if (dataFormItem.FieldName == "Name")
+        {
+            var list = new List<string>();
+            list.Add("Home");
+            list.Add("Food");
+            list.Add("Utilities");
+            list.Add("Education");
+            (dataFormItem as DataFormComboBoxItem).ItemsSource = list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Loading complex type property values in combo box editor
+
+You can display the complex type property values in combo box editor by using the [GetSource]`Link` method of IDataFormSourceProvider class, which is used to get source list as complex property values for combo box editor and set it to `IDataFormSourceProvider` property of SfDataForm. You need to use `GenerateDataFormItem `event to set [DisplayMemberPath]`Link` and [SelectedValuePath]`Link` property value of DataFormComboBoxItem for complex type property.
+
+N> Class cannot be directly set as data type for combo box editor in this complex type scenario.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.ItemsSourceProvider = new SourceProviderExt();
```

Comment:
SourceProviderExt = rename this to DataFormItemsSourceProvider

Rename all uses

Reply:
SourceProviderExt to DataFormItemsourceProvider changed